aiopg/utils.py

```diff
@@ -231,3 +233,10 @@ def __exit__(self, *args):
         coroutines._COROUTINE_TYPES += (_ContextManager,)
     except:
         pass
+
+
+# map psycopg2cffi to psycopg2, so any code
+# that imports psycopg2 will use psycopg2cffi
+if PY_IMPL == 'PyPy':
+    from psycopg2cffi import compat
+    compat.register()
```

Comment:
I have never used `psycopg2cffi`, but what happened if `compat.register()` called two times? or after first import of `psycopg2` in other module?